lib/testScenario/reportGenerator.ts

```diff
@@ -409,7 +409,7 @@ export class ReportGenerator {
       detail: this.dataMasker.jsonStringify(it.response.body),
     };
     const responseObj = this.dataMasker.jsonParse(it.response.body);
-    ret.code = "RUNTIME_ERROR";
+    ret.code = `RUNTIME_ERROR_${it.response.statusCode}`;
     ret.message = responseObj?.error?.message;
```

Comment:
Can you set message = code:${error?.code}, message:${error?.message}

Reply:
Updated.